dynamic/tests/test_get_api.py

```diff
@@ -0,0 +1,23 @@
+from io import StringIO
+import json
+from ..api_test import ApiTesting
+
+
+class TestApi():
+
+    def test_get_request_no_headers(self, monkeypatch):
+        # Test to check get api response without header
+
+        self.test_object = ApiTesting()
+
+        test_uri = "https://reqres.in/api/users/2"
```

Comment:
A test server can also be implemented so that these tests can run successfully without internet connectivity.